src/components/editMeasure/details/EditMeasureDetailsSideNav.tsx

```diff
@@ -25,7 +25,7 @@ export default function EditMeasureDetailsSideNav(
   };
   return (
     <OuterWrapper>
-      <InnerWrapper>
+      <InnerWrapper id="edit-measure-details-side-nav">
```

Comment:
I don't understand, the story talks about pagination component is not as expected ( I am guessing that is the buttons with page numbers at the bottom) But this is fixing Measure-Details Side Nav. How are they related ?

Reply:
@RohitKandimalla 

The reason that the pagination component has the offset spacing is because it's taking a non specific css rule from the side nav that happens to have the same class.

Adding an id to the containing element, and modifying the css for that component to only select children of that ID fixes the issue

Reply:
![image](https://github.com/MeasureAuthoringTool/madie-measure/assets/84744653/010dab8c-8278-4354-b052-42768690c862)